backend/src/main/scala/bloop/util/Base64Uuid.scala

```diff
@@ -0,0 +1,27 @@
+package bloop.util
+import java.nio.ByteBuffer
+import java.util.{Base64, UUID}
+
+object Base64Uuid {
+
+  // Copied from https://stackoverflow.com/a/29836273
+  implicit class UuidWithBase64(val uuid: UUID) extends AnyVal{
+    def toBase64String: String = {
+      val bb = ByteBuffer.wrap(new Array[Byte](16))
+      bb.putLong(uuid.getMostSignificantBits)
+      bb.putLong(uuid.getLeastSignificantBits)
+      Base64.getUrlEncoder.encodeToString(bb.array)
+    }
+  }
+
+  def apply(base64: String): UUID = {
+    val bytes = Base64.getUrlDecoder.decode(base64)
+    val bb = ByteBuffer.wrap(bytes)
+    val high = bb.getLong
+    val low = bb.getLong
+    new UUID(high, low)
+  }
+
+  def randomUUID(): String = UUID.randomUUID.toBase64String
```

Comment:
`def randomUUID: String` without `()` is more idiomatic

Reply:
Ok